docs/source/changes.rst

```diff
@@ -3,18 +3,35 @@ History of changes
 
 .. currentmodule:: h11
 
-v0.8.1 (2018-04-14)
+v0.9.0 (2019-04-22)
 -------------------
 
 Bug fixes:
 
-* Always return headers as ``bytes`` objects (`#60
-  <https://github.com/python-hyper/h11/issues/60>`__)
 * Allow a broader range of characters in header values. This violates
   the RFC, but is apparently required for compatibility with
   real-world code, like Google Analytics cookies (`#57
   <https://github.com/python-hyper/h11/issues/57>`__, `#58
   <https://github.com/python-hyper/h11/issues/58>`__)
+* Constrain regex request-target matching (`#69
+  <https://github.com/python-hyper/h11/pull/69>`__)
```

Comment:
This is a pretty terse description, for an important change :-)

Maybe something like:

Validate incoming and outgoing request paths for invalid characters. This prevents a variety of potential security issues that have affected other HTTP clients.

Reply:
:+1: